src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -74,18 +72,22 @@ export default {
     },
     totalRewards() {
       if (this.rewards && this.rewards.length > 0) {
-        const stakingDenomRewards = this.rewards
-          .filter(({ denom }) => denom === this.stakingDenom)
-          .reduce((sum, { amount }) => sum + Number(amount), 0)
-          .toFixed(6)
-        // if staking denom rewards are not above zero, we'd display the fees and amount for the first existing
-        // alt-token reward on the rewards array
-        return stakingDenomRewards > 0
-          ? stakingDenomRewards
-          : this.rewards
-              .filter(({ denom }) => denom === this.feeDenom)
-              .reduce((sum, { amount }) => sum + Number(amount), 0)
-              .toFixed(6)
+        return this.rewards.reduce(
+          (totalRewardsAgreggator, { amount, denom }) => {
+            let rewardDenom = denom
+            let sameDenomReward = totalRewardsAgreggator.find(
+              ({ denom }) => denom === rewardDenom
+            )
+            sameDenomReward
```

Comment:
please use if/then if you do an action. only use the ternary operator if you do assignments